components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/java/org/wso2/carbon/mediator/enrich/ui/EnrichMediator.java

```diff
@@ -28,12 +27,15 @@
 import org.wso2.carbon.mediator.service.MediatorException;
 import org.wso2.carbon.mediator.service.ui.AbstractMediator;
 
+import java.util.Iterator;
 import javax.xml.namespace.QName;
-import javax.xml.stream.XMLInputFactory;
 import javax.xml.stream.XMLStreamException;
-import javax.xml.stream.XMLStreamReader;
-import java.util.Iterator;
 
+/**
+ * It represent the attributes and functions of enrich mediator, process a message based on a given
+ * source configuration and then perform the specified action on the message by using the
+ * target configuration
+ */
 
```

Comment:
If you keep a space between doc comment and class definition, It will not be picked up the class doc comment. Therefore better to remove the new line.

Reply:
Done